src/utils/parse-raw-flags.js

```diff
@@ -17,7 +17,7 @@ const parseRawFlags = function (raw) {
       const next = array[index + 1]
       if (!next) {
         acc[key] = true
-      } else if (next && next && /^-{1,2}/.test(next)) {
+      } else if (/^-{1,2}/.test(next)) {
```

Comment:
This could probably be refactored to `next.startsWith('--') || next.startsWith('-')`, not sure it's better though.